driver/index.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
-const eventEmitter = require('../eventPool');
-const { handlePickupAndDelivery } = require('./handler');
+const { io } = require('socket.io-client');
+const socket = io('http://localhost:3001/caps');
+const { handlePickupAndDelivered } = require('./handler');
 
-eventEmitter.on('pickup', handlePickupAndDelivery);
\ No newline at end of file
+socket.on('pickup', handlePickupAndDelivered);
+
+let store = '1-206-flowers';
+socket.emit('JOIN', store);
```

Comment:
when you join a room here, only this socket is in the room and the handler socket is not. It's not really noticeable here, because you're not doing anything with the room other than joining it, but you need to be careful. If you want to have the handler functions in a separate file, you need to pass it the socket with a curried callback function, e.g.
``` javascript
socket.on('pickup', (payload) => handlePickupAndDelivery(payload, socket));
```